tools/replay/replay_settings.h

```diff
@@ -75,11 +76,16 @@ const char kScreenshotRangeArgument[]          = "--screenshots";
 const char kScreenshotFormatArgument[]         = "--screenshot-format";
 const char kScreenshotDirArgument[]            = "--screenshot-dir";
 const char kScreenshotFilePrefixArgument[]     = "--screenshot-prefix";
+const char kMeasurementRangeArgument[]         = "--measurement-frame-range";
+const char kQuitAfterMeasurementRangeOption[]  = "--quit-after-measurement-range";
+const char kFlushMeasurementRangeOption[]      = "--flush-measurement-range";
 
 const char kOptions[] = "-h|--help,--version,--log-debugview,--no-debug-popup,--paused,--sync,--sfa|--skip-failed-"
-                        "allocations,--opcd|--omit-pipeline-cache-data,--remove-unsupported,--screenshot-all";
+                        "allocations,--opcd|--omit-pipeline-cache-data,--remove-unsupported,--screenshot-all"
```

Comment:
There needs to be a `,` after `--screenshot-all` so it is recognized as distinct from the next switch.

Right now this is interpreted as a single switch `--screenshot-all--qamr`.

I'm also concerned that testing didn't exercise the new `--qamr` switch (which would have caught this issue).

Reply:
Sorry about that, fixed now!

Want me to look into the testing as well (i guess at the very least, raising a new issue would be a good idea)?

Reply:
I agree.  Certainly automated testing needs to be added.  But the implementer here should have manually tested with the new options to make sure they work.  It's alarming to realize that this didn't occur.

I'd like an issue created that includes the manual test plan and the results of executing that plan.  The manual test plan so documented can then be used on both our side and yours to add automated tests.

Reply:
> But the implementer here should have manually tested with the new options to make sure they work.
Absolutely, i forgot about the --qamr and --fmr shortened options. Wont happen again.

Here is the issue, this should cover all the edge cases: https://github.com/LunarG/gfxreconstruct/issues/561

Reply:
Any more feedback on this? Its a pretty important feature on our end so would be nice if we could get it merged (or get some more pointers on what needs to be done to get it merged)

Reply:
I appreciate the issue created.  I think @lunarpapillo 's concern is that it didn't look like you had tested the changes you had made.  It's the responsibility of the PR submitter to test the code before we'll accept it.  Please note @lunarpapillo  said "I'd like an issue created that includes the manual test plan **and the results of executing that plan**." (my emphasis)  We will put a request for automated testing into our plan, but more important is whether you tested yourself with those options you've added, as Bob suggests?

Reply:
> but more important is whether you tested yourself with those options you've added, as Bob suggests?
I did yes, it works fine now :)

I ran all the tests listed in the issue before submitting the pull request, but i always used the full length arguments, eg. --measurement-frame-range instead of --mfr, the bug was only exposed when you used the short version (eg. --mfr would break, --measurement-frame-range would not) so i assumed everything to be ok which was a mistake (essentially, i forgot to test one part of the argument parser code, i was too focused on the "internal" code i changed).